### PR TITLE
Add getResourceCustomized View REST API

### LINKED ISSUE #1425: Add REST API for get resource customized view
**Is your feature request related to a problem? Please describe.**
We need a REST API for getting resource customized view for our users.

**Describe the solution you'd like**
Add a REST API.

**Additional context**

### PR DESCRIPTION
**Pull request by zhangmeng916**
### Issues

- [X] My PR addresses the following Helix issues and references them in the PR description:

Fixed #1425 
### Description

- [X] Here are some details about my PR, including screenshots of any UI changes:

We need a REST API for getting resource customized view for our users. This PR adds one in resource accessor.

### Tests

- [X] The following is the result of the "mvn test" command on the appropriate module:

helix-core
[INFO]
[INFO] Results:
[INFO]
[WARNING] Tests run: 1210, Failures: 0, Errors: 0, Skipped: 1
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:07 h
[INFO] Finished at: 2020-10-02T14:57:53-07:00
[INFO] -----------------------------------------

helix-rest
[INFO] Tests run: 169, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 122.996 s - in TestSuite
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 169, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:09 min
[INFO] Finished at: 2020-10-02T09:20:49-07:00
### Documentation (Optional)

- In case of new functionality, my PR adds documentation in the following wiki page:

(Link the GitHub wiki you added)

### Commits

- My commits all reference appropriate Apache Helix GitHub issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Code Quality

- My diff has been formatted using helix-style.xml 
(helix-style-intellij.xml if IntelliJ IDE is used)

### CODE REVIEW

File: helix-rest/src/test/java/org/apache/helix/rest/server/TestResourceAccessor.java [hunk 1]
```diff
@@ -162,6 +162,13 @@ public void testExternalView() throws IOException {
     System.out.println("End test :" + TestHelper.getTestMethodName());
   }
 
+  @Test(dependsOnMethods = "testExternalView")
+  public void testCustomizedView() {
+    System.out.println("Start test :" + TestHelper.getTestMethodName());
+    get("clusters/" + CLUSTER_NAME + "/resources/" + RESOURCE_NAME + "/customizedView", null,
+        Response.Status.NOT_FOUND.getStatusCode(), true);
+  }
+
```
Comment (mgao0) [inline helix-rest/src/test/java/org/apache/helix/rest/server/TestResourceAccessor.java:171]: Could you add a test case where the call returns something?

Comment (mgao0) [reply helix-rest/src/test/java/org/apache/helix/rest/server/TestResourceAccessor.java:182]: Thanks for adding the test.

**Review by mgao0 — APPROVED** [review]
LGTM

**Comment by zhangmeng916** [comment]
This PR is ready to merge. Approved by @mgao0 
Final commit message:
Add getResourceCustomized View REST API

**Pull request merged**